doc/classes/InputEventJoypadButton.xml

```diff
@@ -4,6 +4,7 @@
 		Input event type for joypad button events.
 	</brief_description>
 	<description>
+		Contains joypad button informations. See [method Node._input] and http://docs.godotengine.org/en/stable/learning/features/inputs/inputevent.html?highlight=input#inputevent.
```

Comment:
`Input event type for gamepad buttons. For joysticks see [InputEventJoypadMotion].`